src/etc/default/vyatta

```diff
@@ -0,0 +1,203 @@
+# declare configured Vyatta shell environment variables
```

Comment:
I think we should likely add a license header with attribution to Vyatta, to avoid creating an impression that we wrote us, and to ensure legal compliance.

Reply:
Of course - attribution added